allennlp/modules/elmo.py

```diff
@@ -75,6 +76,7 @@ def __init__(self,
                  requires_grad: bool = False,
                  do_layer_norm: bool = False,
                  dropout: float = 0.5,
+                 vocab_to_cache: List[str] = None,
```

Comment:
Add this to the docstring.  Also, it looks like passing this changes the expected input to the function, yes?  Is that documented anywhere?